app/controllers/responsiblepeople/TrainingController.scala

```diff
@@ -77,10 +77,7 @@ class TrainingController @Inject()(
       case Some(cacheMap) => {
         (edit, cacheMap.getEntry[BusinessMatching](BusinessMatching.key)) match {
           case (true, _) => Redirect(routes.DetailedAnswersController.get(index, flow))
-          case (false, Some(BusinessMatching(_, Some(BusinessActivities(acts, _, _, _)),_,_,_,_, _, _, _)))
```

Comment:
the check on acts.exist(...) in no longer used.
also the original code redirect to 2 controllers, now its only one.
is this intentional? as to me it seems to have changed the code flow

Reply:
I did it intentionally because after training question in non-edit mode user should go to FitAndProperNotice regardless of business type. Is it right?

Reply:
I think because we always ask fit and proper and always show the notice this refactor is ok?